frontend/js/src/utils/APIService.ts

```diff
@@ -588,8 +588,17 @@ export default class APIService {
       throw new SyntaxError("Username missing");
     }
 
-    const url = `${this.APIBaseURI}/feedback/user/${userName}/get-feedback-for-recordings?recording_msids=${recording_msids}&recording_mbids=${recording_mbids}`;
-    const response = await fetch(url);
+    const url = `${this.APIBaseURI}/feedback/user/${userName}/get-feedback-for-recordings`;
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json;charset=UTF-8",
+      },
+      body: JSON.stringify({
```

Comment:
I see there are a few different places where this method is being called. I have two approaches in mind. 
1. Create a util function to modify the callers of this APIService method to pass an array instead of a string.
2. Create a util function in the APIService method itself to convert this list of strings into an array and then make a request to the server.

Which approach should I go ahead with?

Reply:
As far as I know, we manually convert arrays to comma separated string before passing to this method. So you should change every place its called to not create that string and instead pass an array: For one example:

https://github.com/metabrainz/listenbrainz-server/blob/b8c86e290bb7427f005e441eba951ce0cbd2e6f5/frontend/js/src/pins/UserPins.tsx#L131-L148

Reply:
That makes sense, thanks